docs/index.md

```diff
@@ -25,17 +25,19 @@ You must **legally** obtain your own ROM. This site does not endorse or condone
 
 Open `Project64KSE.exe` and select your ROM directory (where your ROM is located).
 
+![rom directory selection](/assets/images/rom_directory.png)
 
 ## Select your Plugins
 
+Plugins are selected in `Options -> Settings -> Plugins`. You can configure each plugin through `Options -> Configure Plugin`.
 
 ### Graphics
 
-#### GLideN64 Public Release 4.0 `Recommended`{ .badge data-md-color-primary="green" } `OpenGL`{ .badge data-md-color-primary="cyan" } { #glide4 data-toc-label='GLideN64 4.0' }
+#### GLideN64 Public Release 4.0 `OpenGL`{ .badge data-md-color-primary="cyan" } { #glide4 data-toc-label='GLideN64 4.0' }
 
 The most accurate and best looking plugin, but usually crashes the emulator when ending emulation.
 
-#### GLideN64 Public Release 2.0 `OpenGL`{ .badge data-md-color-primary="cyan" } { #glide2 data-toc-label='GLideN64 2.0' }
+#### GLideN64 Public Release 2.0 `Recommended`{ .badge data-md-color-primary="green" } `OpenGL`{ .badge data-md-color-primary="cyan" } { #glide2 data-toc-label='GLideN64 2.0' }
```

Comment:
As much as I dislike having the older plugin be the recommended one, I agree it should be v2 at this point. But we should have the recommended plugin at the top, above v4.

Reply:
Fixed in https://github.com/smash64-dev/smash64-online/pull/10/commits/da19be5987fa6d2f4230dea0e60b1096e06ed60f.